apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-save-click.js

```diff
@@ -7,12 +9,26 @@ import tracksRecordEvent from './track-record-event';
  */
 export const wpcomBlockEditorSaveClick = () => ( {
 	id: 'wpcom-block-editor-save-click',
-	// The first selector is for site editing. The second is for post editing.
 	selector:
 		'.editor-entities-saved-states__save-button, .editor-post-publish-button:not(.has-changes-dot)',
 	type: 'click',
 	handler: () => {
-		tracksRecordEvent( 'wpcom_block_editor_save_click' );
+		const isSiteEditor = getEditorType() === 'site';
+		const isCurrentPostPublished = select( 'core/editor' ).isCurrentPostPublished();
+		const isEditedPostBeingScheduled = select( 'core/editor' ).isEditedPostBeingScheduled();
+		let actionType = 'publishing';
+
+		if ( isSiteEditor ) {
+			actionType = 'saving';
+		} else if ( isEditedPostBeingScheduled ) {
+			actionType = 'scheduling';
+		} else if ( isCurrentPostPublished ) {
+			actionType = 'updating';
```

Comment:
Slight preference from me for 'publish', 'save', 'schedule', 'update'; somehow sounds better for action type without the "-ing"... but that could just be me!